core/trino-web-ui/src/main/resources/webapp/src/components/WorkerStatus.jsx

```diff
@@ -267,9 +267,9 @@ export class WorkerStatus extends React.Component {
             <div>
                 <table className="table">
                     <tbody>
-                        {Object.keys(queries).map((key) =>
-                            WorkerStatus.renderPoolQuery(key, queries[key][0], queries[key][1], size)
-                        )}
+                        {Object.keys(queries)
```

Comment:
You can enumerate `Object.values(queries)` to avoid having to look-up in the rest of the code

Reply:
Key (= query_id) is also required for rendering. Sure we can convert `Map[QueryId, (Reserved, Revocable)]` to `Array[(QueryId, Reserved, Revocable)]` first to avoid later lookup. Is this what you mean?